frontend/device/input.lua

```diff
@@ -298,6 +299,13 @@ function Input:handleKeyBoardEv(ev)
         end
     end
 
+    -- quit on Alt + F4
+    -- this is also emitted by the close event in SQL
+    if self.modifiers["Alt"] and keycode == "F4" then
+        local UIManager = require("ui/uimanager")
+        UIManager:quit()
```

Comment:
I think this will exit UIManager without saving settings (looks like filemanagermenu.lua and readermenu.lua do other things in their :exitOrRestart(callback)).
May be a bit complicated to reach their proper exit methods from there...

Reply:
True, but this is just intended for development as a (sometimes) more convenient form of Ctrl+C in the terminal. Real exit would be a little more complicated.

https://github.com/koreader/koreader/blob/177485f84f1bcc1fd28c73d9bf6de91d384acdc7/frontend/apps/reader/modules/readermenu.lua#L220-L232

https://github.com/koreader/koreader/blob/177485f84f1bcc1fd28c73d9bf6de91d384acdc7/frontend/apps/filemanager/filemanagermenu.lua#L302-L328

Reply:
OK then. May be just add a ``logger.warn("Quick exit, settings not saved.")``